files/en-us/glossary/clickjacking/index.html

```diff
@@ -0,0 +1,26 @@
+---
+title: Clickjacking
+slug: Glossary/Clickjacking
+tags:
+  - Clickjacking
+  - Interface-based attack
+  - Glossary
+  - Security
+  - vulnerability
+  - exploit
+---
+<p>Clickjacking is an interface-based attack that tricks website users into unwarily clicking on malicious links. In clickjacking, the attackers embed their malicious links into buttons or legitimate pages in a website. In an infected {{glossary("Site")}}, whenever a user clicks on a legitimate link, the attacker gets the confidential information of that user - which ultimately compromises the user's privacy on the Internet. Clickjacking falls under the <a href="https://www.owasp.org/index.php/Top_10-2017_A6-Security_Misconfiguration">A6 – Security Misconfiguration</a> item in OWASP’s 2017 Top 10 list.</p> 
+
+<p>Clickjacking can be prevented by implementing a Content Security Policy (CSP) and <a href="https://cheatsheetseries.owasp.org/cheatsheets/Clickjacking_Defense_Cheat_Sheet.html">other defense methods</a>.</p>
+
+<h2 id="Learn_more">Learn more</h2>
+
+<h3 id="General_knowledge">General knowledge</h3>
+
+<ul>
+ <li><a href="/en-US/docs/Web/Security/Types_of_attacks#clickjacking">Clickjacking</a></li>
+ <li><a href="https://en.wikipedia.org/wiki/Clickjacking">Clickjacking</a>on Wikipedia</li>
+ <li><a href="https://owasp.org/www-community/attacks/Clickjacking">Clickjacking on OWASP</a></li>
```

Comment:
For consistency with the previous item

```suggestion
 <li><a href="https://owasp.org/www-community/attacks/Clickjacking">Clickjacking</a> on OWASP</li>
```